app/Jobs/ValidateCSV.php

```diff
@@ -136,6 +137,10 @@ private function checkFieldErrors($mismatch): ?string
                 'prohibited_if:wikidata_value,',
                 'max:' . $rules['meta_wikidata_value']['max_length'],
                 'regex:' . $rules['meta_wikidata_value']['format']
+            ],
+            'type' => [
+                'string',
+                Rule::in(['statement', 'qualifier']),
```

Comment:
That's great, thank you! I think we should keep these accepted values in the config, like all the other rules in [config/mismatches.php](https://github.com/wmde/wikidata-mismatch-finder/blob/development/config/mismatches.php).

Reply:
:grimacing: